src/sections/Home/Partners-home/partnerSection.style.js

```diff
@@ -1,103 +1,94 @@
 import styled from "styled-components";
 const PartnerItemWrapper = styled.section`
-    padding: 2rem 0;
-    margin: 0rem 0rem 5rem 0rem;
-    overflow: hidden;
-    .section-title{
-        h4{
-            color: ${props => props.theme.grey737373ToGrey4C4A4A};
-            text-align: center;
-            margin-top: .5rem;
-            transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
-        }
-    }
-    .partner-image{
-        margin-left:1.5rem;
-        margin-right:1.5rem;
+  padding: 2rem 0;
+  margin: 0rem 0rem 5rem 0rem;
+  overflow: hidden;
+  .section-title {
+    h4 {
+      color: ${(props) => props.theme.grey737373ToGrey4C4A4A};
+      text-align: center;
+      margin-top: 0.5rem;
+      transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
     }
-    .partner-slider{
-        display: flex;
-        justify-content: center;
-        flex-wrap: nowrap;
-        @media(max-width: 1400px){
-            flex-wrap: wrap;
-        }
+  }
+  .partner-image {
+    margin-left: 1.5rem;
+    margin-right: 1.5rem;
+  }
+  .partner-slider {
+    display: flex;
+    justify-content: center;
+    flex-wrap: nowrap;
+    @media (max-width: 1400px) {
+      flex-wrap: wrap;
     }
-		.slick-dots li button:before{
-			color: ${props => props.theme.grey939393ToGrey4C4A4A}
-            transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
-		}
-		.slick-dots li.slick-active button:before {
-			color: ${props => props.theme.whiteToGrey4C4A4A};
-            transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
-		}
-		.slick-dots li button:hover{
-				box-shadow: none;
-		}
-
-    a.partner-card {
-        @media(max-width: 1400px){
-            flex: 0 0 12%;
-            margin-left:0.5rem;
-            margin-right:0.5rem;
-        }
-
-        &:hover
-        {
-            img{
-                opacity: 1;
-            }
-        }
+  }
+  .slick-dots li button:before {
+    color: ${(props) => props.theme.grey939393ToGrey4C4A4A};
+    transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
+  }
+  .slick-dots li.slick-active button:before {
+    color: ${(props) => props.theme.whiteToGrey4C4A4A};
+    transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
+  }
+  .slick-dots li button:hover {
+    box-shadow: none;
+  }
 
-        #VMware:hover
-        {
-        
-           filter:brightness(10%);
-        
-
-         }
+  a.partner-card {
+    @media (max-width: 1400px) {
```

Comment:
I am not able to understand why you have written two @media queries for the same max-width i.e 1400px. 
You can write under one media query for same width. 
Or please correct me where I am wrong.

Reply:
Actually it was like that when I clone it so I am not sure why it is like that.